### ISSUE TITLE
(Android) On Property changed not fired when carousel view is inside ViewCell (Listview)

### ISSUE THREAD

**Issue by blattepop** [alexrainman/CarouselView#183]
I am trying to do something when the position of a carousel is changed. So on the CarouselViewControl.PropertyChanged I try to run this code : 
`if (e.PropertyName == "Position")
{
 ; // Do something
}`
However, the PropertyChanged event is only fired on iOS and not on Android.

Simple repro code of the page in attachment.

[ListViewSwipePageTest.zip](https://github.com/alexrainman/CarouselView/files/1174641/ListViewSwipePageTest.zip)

**Comment by alexrainman** [comment]
Why are you doing this yourself? This is internaly implemented already. If you want to know when position changes, use PositionSelected delegate.

**Comment by blattepop** [comment]
Right I missed that completely because I am using Xaml and it does not show up on Xaml Intellisense.

Thanks for the heads up and very quick response!

**Comment by alexrainman** [comment]
Yeah, as it's an EventHandler, so i can Invoke it from renderers, it doesn't show up in XAML.

**Comment by blattepop** [comment]
What I ended up doing is subclassing your control and add this so I can set it up in xaml.

public class BaseCarouselView : CarouselViewControl
    {
        public event EventHandler<int> PositionChanged;

        public BaseCarouselView()
        {
            PositionSelected = LocalPositionChanged;
        }

        private void LocalPositionChanged(object sender, int i)
        {
            if (PositionChanged != null)
                PositionChanged(sender, i);
        }
    }

However the LocalPositionChanged is never fired on Android as well...

**Comment by alexrainman** [comment]
Why you need to subclass? Also, there's some issues with Carousel and ListView. Take a look at the other opened issues.

**Comment by blattepop** [comment]
Hi !
I fixed the issue by getting the plugin from your source instead of Nuget and commenting this code at line 55 :
`
				/*
if (viewPager != null)
				{
					viewPager.PageSelected -= ViewPager_PageSelected;
					viewPager.PageScrollStateChanged -= ViewPager_PageScrollStateChanged;
				}
*/
` 
I've seen that the viewPager is cleaned in CleanUpViewPager anyway, so we don't really need this code. What happened is that the code above was called after SetNativeView() so that was removing my event.

I guess it's an issue that only appears in listviews but I don't think we really need the code I commented out.
